src/flask_pyoidc/auth_response_handler.py

```diff
@@ -49,7 +49,7 @@ def __init__(self, client):
         """
         self._client = client
 
-    def process_auth_response(self, auth_response, auth_request):
+    def process_auth_response(self, auth_response, auth_request, **kwargs):
```

Comment:
Since the `**kwargs` are intended to be forwarded to the token request, I think it makes more sense for this to be a named parameter. Something like `extra_token_params` (with type `Mapping[str, Any]`).
This parameter should also be added to the docstring for this method.